### PR TITLE
Fix cypress dependencies

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

UPDATE: The broken dependency issue seems to have been fixed upstream. This PR still might be worth so we don't have unnecessary `devDependencies` installed.
- This PR also fixes a broken test and improves the output of `formatLabel()` utility.

~Today all frontend-related CI runs started failing (could be due to this, https://github.com/cypress-io/cypress/issues/15797).~

- This PR removes @cypress dependencies from main package.json in favor of installing them directly in the CI runs (as we do with the main cypress package).
- We also target a specific version of these dependencies that works fine.

## Checklist

- [X] All querysets/queries filter by Organization, by Team, and by User. **Not applicable.**
- [X] Django backend tests. **Not applicable.**
- [X] Jest frontend tests
- [X] Cypress end-to-end tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,8 +90,6 @@
         "@babel/plugin-transform-runtime": "^7.10.4",
         "@babel/preset-env": "7.10.4",
         "@babel/preset-typescript": "^7.10.4",
-        "@cypress/react": "^4.16.4",
-        "@cypress/webpack-preprocessor": "^5.5.0",
```
Comment (EDsCODE) [inline package.json:94]: good call

**Review by EDsCODE — APPROVED** [review]
This looks good now. There seems to be a flaky backend test though

**Comment by paolodamico** [comment]
Failing backend test is unrelated. Merging https://github.com/PostHog/posthog-cloud/pull/100/ will fix it.

**Comment by EDsCODE** [comment]
Hmm I just realized though that we're going to need some solution for running cypress locally

**Comment by paolodamico** [comment]
Well I can whip up a simple script. Usually what I do is just copy the command on the GH script and run it.

**Pull request merged**